controller/tap/handlers.go

```diff
@@ -363,3 +343,38 @@ func renderJSONError(w http.ResponseWriter, err error, status int) {
 	w.WriteHeader(status)
 	w.Write(rsp)
 }
+
+type serverStream struct {
```

Comment:
The implementation here reflects a previous revision of the tap server in [stable-2.4.0](https://github.com/linkerd/linkerd2/blob/53e589890ddb0283be794e71ebe7cba250b02a11/controller/api/public/http_server.go#L317-L318)